src/pubsub-engine.ts

```diff
@@ -1,5 +1,5 @@
 export interface PubSubEngine {
-  publish(triggerName: string, payload: any): boolean;
+  publish(triggerName: string, payload: any): Promise<void>;
```

Comment:
We should document this change in CHANGELOG

Reply:
Got it, I assume we should bump to 0.6.0?

Reply:
I put this under vNext, right?

Reply:
Yup.